packages/notifi-react-card/lib/context/NotifiFormContext.tsx

```diff
@@ -24,18 +25,13 @@ export type NotifiFormData = Readonly<{
   setTelegramErrorMessage: (value: string) => void;
```

Comment:
Should the type of the value be DestinationError here?

Reply:
for now, no. this is in the formContext so it's handling it a little more simply compared to the subscription context